win32ss/gdi/gdi32/objects/bitmap.c

```diff
@@ -396,66 +424,66 @@ GetDIBits(
 HBITMAP
 WINAPI
 CreateDIBitmap(
-    HDC hDC,
```

Comment:
Don't make these changes please. It makes it incredibly hard to review this PR.

Reply:
I'm confused. How does it make it _incredibly_ hard? It should hardly require review at all, since it's just a find & replace inside a single method. Plus it's relegated to a separate `[FORMATTING]` commit and thus has no functional changes – as recommended in the style guide.

Reply:
There are variables being renamed in https://github.com/reactos/reactos/pull/1381/commits/d3908a527a211ad60c4e907492c1ed5db85a6509
And it makes it hard to spot changes that aren't related to the renamed variables, i.e. to find the logic changes.

Reply:
Now that I can understand. Fixed.